### ISSUE TITLE
[ENH] Improve "Unrecognized provider" warning more

### ISSUE THREAD

**Issue by verbunk** [serverless/serverless#9017]
### Use case description

⚠️⚠️ REPLACE WITH DESCRIPTION

**Comment by medikoo** [comment]
@verbunk thanks for proposal. Still `Provider functionality warning` is also slightly misleading, as issue is not about how _provider_ functions, but purely that it doesn't provide configuration schema for its config.

I guess some improvement might be if we reword it to: `Configuration warning:  Unspecified provider 'openwhisk'`, and then following sentences I think really well explain what that means, it's just question of reading it :)
